mantis/cli/stitch.py

```diff
@@ -0,0 +1,197 @@
+import datetime
+import shutil
+
+from pathlib import Path
+
+import click
+import numpy as np
+import pandas as pd
+
+from iohub import open_ome_zarr
+from iohub.ngff_meta import TransformationMeta
+from slurmkit import SlurmParams, slurm_function, submit_function
+
+from mantis.analysis.AnalysisSettings import StitchSettings
+from mantis.analysis.stitch import (
+    get_grid_rows_cols,
+    get_image_shift,
+    get_stitch_output_shape,
+    preprocess_and_shift,
+    stitch_shifted_store,
+)
+from mantis.cli.parsing import config_filepath, input_position_dirpaths, output_dirpath
+from mantis.cli.utils import create_empty_hcs_zarr, process_single_position_v2, yaml_to_model
+
+
+@click.command()
+@input_position_dirpaths()
+@output_dirpath()
+@config_filepath()
+@click.option(
+    "--temp-path",
+    type=click.Path(exists=True, file_okay=False, dir_okay=True),
+    default='/hpc/scratch/group.comp.micro/',
```

Comment:
Hmmm I didn't have default write access here. Do you know if there's an everyone-accessible `/tmp/`? Or could the default be `./`?

Doesn't need to be solved now...for now I'll suggest adding `--temp-path` to the docstring.

Reply:
That's quite strange - I thought all of CompMicro should have access to `/hpc/scratch/group.comp.micro/`. Maybe check with IT? It's possible that you were trying to overwrite one of my temp files?

It's important that all jobs have access to this directory, that's why `/tmp` on our HPC is not a good choice since that space is local and different per node. As you suggest, `./` may be a better more general default, with a suggestion to use `/hpc/scratch/group.comp.micro/` in the docstring

Reply:
Aha! My mistake. 

I just looked closer and found that you own `/TEMP_A3_600k_38x38_1.zarr`, so when I copied/reran your scripts I got permissions errors from that directory, not `/hpc/scratch/group.com.micro/`.